docs/_data/table/en/cluster-management-configuration.yml

```diff
@@ -1,10 +0,0 @@
-alluxio.integration.master.resource.cpu:
```

Comment:
this file is empty now, delete it? as well as its reference in the property list tables

```
$ grep -r "cluster-management-configuration" .                                                                                                                                                [20:59:58]
./cn/reference/Properties-List.md:## 集群管理配置项 {#cluster-management-configuration}
./cn/reference/Properties-List.md:{% for item in site.data.table.cluster-management-configuration %}
./cn/reference/Properties-List.md:    <td>{{ site.data.table.cn.cluster-management-configuration[item.propertyName] }}</td>
./en/reference/Properties-List.md:{% for item in site.data.table.cluster-management-configuration %}
./en/reference/Properties-List.md:    <td>{{ site.data.table.en.cluster-management-configuration[item.propertyName] }}</td>
```